### DISCUSSION TITLE
Welcome to Frappe CRM Discussions!

### DISCUSSION THREAD

**Discussion by shariquerik** [frappe/crm#466] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by metalmon** [comment]
Hi👋 I'm a developer and I really like this CRM. I promote it on my territory and really hope that such discussions will help make this CRM the best among the best!

**Comment by nilpatel42** [comment]
Hi @shariquerik, I want to know about the Data tab. What is its use case?

https://github.com/frappe/crm/pull/465

**Reply by shariquerik** [reply]
Sidepanel is used to show frequently used fields but what if we need more fields also we cannot show child tables in side panel
That's why Data tab was added
Currently it does not support child table but I am working on it
